### ISSUE TITLE
Conteúdo de post quebra margem

### ISSUE THREAD

**Issue by anapaulabrasa** [redelivre/redelivre#72]
Guarani
Na interna do post, o conteúdo quebra a margem direita.

![singlepostinterna](https://cloud.githubusercontent.com/assets/2422926/14464477/4d5ce9b6-00a5-11e6-83f6-9a04ede017ef.png)

**Comment by jacsonp** [comment]
e a versão no ar não faz isso?

**Comment by anapaulabrasa** [comment]
Versão atual que estamos utilizando do WP?
Não... Veja:

![singlepost](https://cloud.githubusercontent.com/assets/2422926/14464985/9733e4b6-00a7-11e6-9429-697fb6f7631d.png)

**Comment by jacsonp** [comment]
seu teste foi diferente, veja o problema é com palavras compridas:
http://testedesign.redelivre.org.br/2016/04/12/teste-teste-longo/
![captura de tela de 2016-04-12 12 14 50](https://cloud.githubusercontent.com/assets/3977652/14465130/30dd3b62-00a8-11e6-9648-55be1cbde010.png)

**Comment by jacsonp** [comment]
recomento o uso de um loren ipsum, tipo esse:
http://mussumipsum.com/

**Comment by anapaulabrasa** [comment]
Entendi. Mas de qualquer forma ele não deveria quebrar, certo?

**Comment by jacsonp** [comment]
certo, mas pode ver pelo teste que fiz no ar que o bug já vem de antes, não foi causado pela atualização, mas vamos deixar aqui para corrigir no futuro. Obrigado

**Comment by ElidiaVidal** [comment]
Erro continua.
![photo_2017-04-07_13-12-12](https://cloud.githubusercontent.com/assets/26146292/24808893/1314778c-1b94-11e7-8858-df0fd62d7376.jpg)

**Comment by eduardozulian** [comment]
Para o `blog-01`, que a @ElidiaVidal apontou aqui, talvez uma simples atualização no CSS do tema já resolva:

```
#content {
    overflow-wrap: break-word;
}
```

Mais sobre o `word-wrap`: https://www.w3schools.com/cssref/css3_pr_word-wrap.asp

Quem sabe o mesmo funcione para o Guarani. @anapaulabrasa, você tem algum exemplo de página onde isso ocorra?
